.eslintrc.js

```diff
@@ -87,6 +87,7 @@ module.exports = {
       extends: ["plugin:testing-library/react"],
     },
   ],
+  ignorePatterns: ["*.typegen.ts"],
```

Comment:
should also prettier ignore those files?

Reply:
Good point! Added.